test/isolation/specs/insert_dropchunks_race.spec

```diff
@@ -0,0 +1,26 @@
+# Race condition between insert and drop_chunks
+
+setup {
+  CREATE EXTENSION IF NOT EXISTS timescaledb;
+  DROP TABLE IF EXISTS conditions;
+  CREATE TABLE conditions (time timestamptz, device int, temp float);
+  SELECT create_hypertable('conditions', 'time', 'device', 2);
+  INSERT INTO conditions VALUES ('2020-01-03 10:30', 1, 32.2);
+}
+
+teardown {
+  DROP TABLE conditions;
+}
+
+session "s1"
+setup		{ BEGIN; SET TRANSACTION ISOLATION LEVEL READ COMMITTED; }
+step "s1a"	{ INSERT INTO conditions VALUES ('2020-01-03 10:30', 3, 33.4); }
+step "s1b" 	{ COMMIT; }
+step "s1c" 	{ SELECT COUNT(*) FROM _timescaledb_catalog.chunk_constraint LEFT JOIN _timescaledb_catalog.dimension_slice sl ON dimension_slice_id = sl.id WHERE sl.id IS NULL; }
```

Comment:
couple words what you are checking here and what the failure modes are would be nice

Reply:
Done.